server/backend/testspg/authorization/test_jwt_validation.py

```diff
@@ -20,33 +17,27 @@
                                assert_error_response,
                                headers)
 from Crypto.PublicKey import RSA
-from mock_alchemy.mocking import UnifiedAlchemyMagicMock
+from testspg.constants import FAM_APPLICATION_ID
+
 
 LOGGER = logging.getLogger(__name__)
-endPoint = f"{apiPrefix}/fam_applications"
+endPoint = f"{apiPrefix}/fam_applications/{FAM_APPLICATION_ID}/user_role_assignment"
 
 
-def test_get_application_success(
-    test_client_fixture_unit: starlette.testclient.TestClient,
+def test_get_application_user_role_assignment_success(
+    test_client_fixture: starlette.testclient.TestClient,
     test_rsa_key
 ):
-    test_client_fixture_unit.app.dependency_overrides[database.get_db] = \
-        lambda: UnifiedAlchemyMagicMock(data=[
-            (
-                [mock.call.query(models.FamApplication), mock.call.all()], []
-            ),
-        ])
-
     token = create_jwt_token(test_rsa_key)
 
-    response = test_client_fixture_unit.get(f"{endPoint}", headers=headers(token))
+    response = test_client_fixture.get(f"{endPoint}", headers=headers(token))
 
     assert response.status_code == 200
     data = response.json()
     assert data == []
 
 
-def test_get_application_no_authentication_failure(
+def test_get_application_user_role_assignment_no_authentication_failure(
```

Comment:
just rename the tests so it makes sense, now we're using the get user role assignment endpoint for testing